gemma-core/src/main/java/ubic/gemma/model/expression/experiment/MetadataType.java

```diff
@@ -0,0 +1,24 @@
+package ubic.gemma.model.expression.experiment;
+
+import ubic.gemma.model.expression.bioAssay.BioAssay;
+
+public enum MetadataType {
+    /**
+     * A sequencing QC report.
+     * <p>
+     * Example: a FastQC report attached to a specific {@link BioAssay}.
+     */
+    SEQUENCING_QC_REPORT,
+    /**
+     * A sequencing alignment report.
+     * <p>
+     * Example: STAR's Log.final.out file on a {@link BioAssay}
+     */
+    SEQUENCING_ALIGNMENT_REPORT,
+    /**
+     * An overall sequencing report.
+     * <p>
+     * Example: a MultiQC report on a {@link ExpressionExperiment}
+     */
+    SEQUENCING_OVERALL_REPORT,
```

Comment:
@ppavlidis I'm looking for some feedback for possible values to put here. If there are things that might make sense to include for microarray platforms like the output of APT tools, etc.

Reply:
There is already some infrastructure to support this (surfaced on diagnostics tab) but it is a low priority now

Reply:
I reviewed it this morning and I synchronized all the MultiQC reports we have generated from the RNA-Seq pipeline.

I also finished adding a new argument to rnaseqDataAdd to take care of copying over the report in Gemma data directory so that it can be integrated fully in the pipeline.